src/pages/tutorial-video-listing/tutorial-video-listing.html

```diff
@@ -0,0 +1,34 @@
+
+<ion-header>
+  <header [title]="'headings.tutorialvideo'|translate" [hideBack]="false">
+  </header>
+</ion-header>
+
+
+<ion-content >
+  <!-- if video presenet -->
+  <div *ngIf="videoList?.length>0">
+    <div *ngFor="let video of videoList; let i = index">    
+      <ion-card>
+        <ion-item text-wrap (click)="playVideo(video.link)">
+          <div class="_flex-box">
+            <ion-icon name="video" class="logoIcon"></ion-icon>   
+            <div style="flex:1">
+              {{video.title}}
+            </div>          
+            <ion-icon name="play" class="menuIcon" ></ion-icon>   
+          </div>
+        </ion-item>
+      </ion-card>
+    </div>
+  </div>
+
+
+  <!-- if video not present -->
+  <div *ngIf="videoList?.length==0">
```

Comment:
use !videoList?.length, its the same as videoList?.length==0

Reply:
changed